core/src/main/java/io/grpc/NameResolver.java

```diff
@@ -120,10 +120,12 @@ public void refresh() {}
      *
      * <p>Implementations will not modify the given {@code servers}.
      *
-     * @param servers the resolved server addresses. An empty list will trigger {@link #onError}
+     * @param servers the resolved server addresses.  Equivalent addresses should be grouped
```

Comment:
Need definition of "equivalent addresses". You probably want to copy the javadoc of `LoadBalancer.handleResolvedAddress()`